app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,83 @@
+'use-strict';
+
+const request = require('request');
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card find-accounts-card">
+				<div class="email-input">
+					<div>Email address(es)</div>
+					<input class="setting-input-value" autofocus placeholder="Add up to 10 comma-separated email addresses"/>
+				</div>
+				<div class="email-input">
+					<button class="green sea w-150" id="find-accounts-button">Find accounts</button>
+				</div>
+			</div>
+		`;
+	}
+
+	confirmation() {
+		return `
+			<div class="sub-title">
+				<div>Emails sent! You will only receive emails at addresses associated with Zulip organizations.</div>
+			</div>
+		`;
+	}
+
+	error() {
+		return `
+			<div class="sub-title">
+				<div>Sorry, looks like something went wrong. Try again?</div>
+			</div>
+		`;
+	}
+
+	init() {
+		this.$findAccounts = this.generateNodeFromTemplate(this.template());
+		this.$confirmation = this.generateNodeFromTemplate(this.confirmation());
+		this.$error = this.generateNodeFromTemplate(this.error());
```

Comment:
I think you should only create the elements when needed, so you should be generating them when the needed below in the code path. Most of the time a user is not going to be using this and when they don't it's very unlikely both of the elements would be needed.